python/mlc_chat/op/attention.py

```diff
@@ -16,12 +16,13 @@
 WARN_FLASHINFER_HEAD_DIM = False
 
 
-def attention(  # pylint: disable=invalid-name,too-many-locals,too-many-statements
+def attention(  # pylint: disable=invalid-name,too-many-locals,too-many-statements,too-many-arguments
     q: nn.Tensor,
     k: nn.Tensor,
     v: nn.Tensor,
     casual_mask: nn.Tensor,
     attn_score_scaling_factor: float = 1.0,
+    attn_weights_out_dtype: str = None,
```

Comment:
I'm a bit skeptical of what `attn_weights` refers to given it can mean many things. How about `qk_dtype`?

Reply:
Makes sense, fixed; thank you!